sqlmesh/core/context.py

```diff
@@ -628,8 +628,16 @@ def format(self) -> None:
                 expressions = parse(
                     file.read(), default_dialect=self.config_for_model(model).dialect
                 )
+                for prop in expressions[0].expressions:
+                    p = t.cast(exp.Property, prop)
```

Comment:
i don't think you need this cast here do you?

Reply:
yea let's remove it, it's not that useful in this context

Reply:
@tobymao done